db/migrate/20240903024211_add_trigger_to_users_to_sync_user_passwords.rb

```diff
@@ -0,0 +1,80 @@
+# frozen_string_literal: true
+class AddTriggerToUsersToSyncUserPasswords < ActiveRecord::Migration[7.1]
+  def up
+    # necessary for postgres < v14 which does not have CREATE OR REPLACE TRIGGER
+    execute <<~SQL.squish
+      DROP TRIGGER IF EXISTS
+      users_password_sync ON users;
+    SQL
```

Comment:
Why do we need to drop the trigger when it is not possible for it to exist before this migration?

Reply:
Icic. I guess no harm running an extra DB statement here.